### PR TITLE
2032: Upgrade tar dependency

### LINKED ISSUE #2032: Upgrade tar dependency to a non vulnerable version
```
# npm audit report

tar  <=3.2.2 || 4.0.0 - 4.4.14 || 5.0.0 - 5.0.6 || 6.0.0 - 6.1.1
Severity: high
Arbitrary File Creation/Overwrite due to insufficient absolute path sanitization - https://npmjs.com/advisories/1770
Arbitrary File Creation/Overwrite via insufficient symlink protection due to directory cache poisoning - https://npmjs.com/advisories/1771
fix available via `npm audit fix`
node_modules/@cloudflare/wrangler/node_modules/tar
```

We actually use this package here: https://github.com/cloudflare/wrangler/blob/8b11ea4664540230cb6970e62306e57ef56a0b28/npm/binary-install.js#L95

I'll be filing a PR to patch/version upgrade.

### PR DESCRIPTION
**Pull request by dhaynespls**
Fixes #2032

### CODE REVIEW

**Comment by jyn514** [comment]
@dhaynespls why did you remove the npmshrinkwrap file?

**Comment by dhaynespls** [comment]
@jyn514 Shouldn't be removed -- just a big diff 
image

**Comment by dhaynespls** [comment]
I specifically ran `npm audit fix` w/ npm version `6.14.5`

File: npm/npm-shrinkwrap.json [hunk 1]
```diff
@@ -1,211 +1,8 @@
 {
   "name": "@cloudflare/wrangler",
-  "version": "1.18.0",
-  "lockfileVersion": 2,
+  "version": "1.19.0",
+  "lockfileVersion": 1,
```
Comment (jyn514) [inline npm/npm-shrinkwrap.json:4]: I think I found why it changed so much - do you have an old version of npm maybe?

**Review by jyn514 — APPROVED** [review]
One nit that's not worth blocking on, let's get this merged :)

File: npm/npm-shrinkwrap.json (hunk 1 above)
Comment (ironheart36925) [reply npm/npm-shrinkwrap.json:4]: Fix me

**Pull request merged**